setup.py

```diff
@@ -37,6 +37,7 @@ def get_advanced_templates():
         'aiohttp==0.22.5',
         'analytics-python',
         'coloredlogs',
+        'dnspython',
```

Comment:
setup cannot handle python module names that do not mostly match their pypi names. See https://mesosphere.atlassian.net/browse/DCOS-11783